test/spawn.jl

```diff
@@ -53,28 +50,30 @@ if valgrind_off
     @test_throws Base.UVError run(`foo_is_not_a_valid_command`)
 end
 
-prefixer(prefix, sleep) = `perl -nle '$|=1; print "'$prefix' ", $_; sleep '$sleep';'`
-@test success(pipeline(`perl -le '$|=1; for(0..2){ print; sleep 1 }'`,
+if is_unix()
+    prefixer(prefix, sleep) = `perl -nle '$|=1; print "'$prefix' ", $_; sleep '$sleep';'`
```

Comment:
can someone explain what this perl is doing? it's been disabled for a long time, ref https://github.com/JuliaLang/julia/commit/e412c69a6e77b0751e0fa0303633e52154937a46, I was the one who turned it back on recently in 373db0f7b510403fd5e22431a830df9be20620ed